include/IECoreScene/MeshAlgo.h

```diff
@@ -101,10 +102,12 @@ IECORESCENE_API std::vector<MeshPrimitivePtr> segment( const MeshPrimitive *mesh
 /// Any PrimitiveVariables that exist will be combined or extended using a default value.
 IECORESCENE_API MeshPrimitivePtr merge( const std::vector<const MeshPrimitive *> &meshes );
 
+const float triangulateDefaultTolerance = 1e-6f;
+
 /// Generate a new triangulated MeshPrimitive
 /// If throwExceptions is true the input mesh is validated to ensure all polygons are convex planar and only then the
 /// tolerance parameter is used to define a floating point epsilon for these checks.
-IECORESCENE_API MeshPrimitivePtr triangulate( const MeshPrimitive *mesh, float tolerance = 1e-6f, bool throwExceptions = false);
+IECORESCENE_API MeshPrimitivePtr triangulate( const MeshPrimitive *mesh, float tolerance = triangulateDefaultTolerance, bool throwExceptions = false, const IECore::Canceller *canceller = nullptr );
```

Comment:
I'm happy to throw these parameters out.  Might make ShrinkWrapOp less useful, but I'm assuming that anything that is still an Op is probably not in active use - and it never actually fixed non-planarity anyway.  I guess shrinkWrap could give inconsistent results now - if we wanted to use shrinkWrap again, we should probabably move it to MeshAlgo, and add a separate testPlanar function it could call.